chat/Makefile.am

```
@@ -0,0 +1,9 @@
+
+sbin_PROGRAMS = chat
+man8_MANS = chat.8
+
+chat_SOURCES = chat.c
+chat_CFLAGS = -DTERMIOS -DSIGTYPE=void -UNO_SLEEP -DFNDELAY=O_NDELAY
```

```suggestion
chat_CPPFLAGS = -DTERMIOS -DSIGTYPE=void -UNO_SLEEP -DFNDELAY=O_NDELAY
```
These are all macro directives

---

Fixed